### ISSUE TITLE
Trouble running halmos after a fresh install

### ISSUE THREAD

**Issue by eugenioclrc** [a16z/halmos#50]
**Describe the bug**
Getting this error after install;
```sh
(base) ➜  curta git:(master) ✗  halmos 
Traceback (most recent call last):
  File "/home/eugenio/.local/bin/halmos", line 5, in <module>
    from halmos.__main__ import main
  File "/home/eugenio/.local/lib/python3.10/site-packages/halmos/__main__.py", line 19, in <module>
    sys.set_int_max_str_digits(0)
AttributeError: module 'sys' has no attribute 'set_int_max_str_digits'
(base) ➜  curta git:(master) ✗ 
```

**To Reproduce**
I have install halmos using `pip3.10 install halmos`, the `pip install halmos` wasnt working for me

**Environment:**
 - OS: Linux ubuntu
 - Python version: [e.g., `python --version`]
 - Halmos and other dependency versions: [e.g., `pip list`]

**Additional context**
I also try creating a venv with no luck :(

**Comment by daejunpark** [comment]
This issue has been fixed in the latest version (0.0.7). Please try reinstalling Halmos and reopen this if the problem still persists.

**Comment by rayeaster** [comment]
> This issue has been fixed in the latest version (0.0.7). Please try reinstalling Halmos and reopen this if the problem still persists.

Great, `0.0.7` works for me while `0.0.6` not
